src/db/prisma/seeders/crawledDataSeeder.ts

```diff
@@ -0,0 +1,71 @@
+import prisma from "../prisma";
+import logger from "../../../utils/logger";
+
+async function crawledDataSeeder() {
+    
+    // Delete all records from crawled data table
+    await prisma.crawledData.deleteMany();
+
+    // Add crawled data into the database
+    await prisma.source.createMany({
```

Comment:
The tables is wrong? Check again. You are seeding crawledData and not source.

Reply:
yes I have missed that